### PR TITLE
Add initial project structure

### PR DESCRIPTION
**Pull request by ayeshLK**
Part of: [#7240](https://github.com/ballerina-platform/ballerina-library/issues/7240)

### CODE REVIEW

File: ballerina/icon.png
Comment (Nuvindu) [inline ballerina/icon.png:1]: Shall we reduce the resolution of the icon here? High resolution image icons may not look better in the Ballerina central

File: ballerina/build.gradle [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+            ignoreExitValue true
+            if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+                commandLine 'cmd', '/c', "git commit -m \"[Automated] Update the toml files\" Ballerina.toml Dependencies.toml"
+            } else {
+                commandLine 'sh', '-c', "git commit -m '[Automated] Update the toml files' Ballerina.toml Dependencies.toml"
+            }
+        }
+    }
+}
+
```
Comment (Nuvindu) [inline ballerina/build.gradle:75]: Can we add the publishing task here?

Comment (ayeshLK) [reply ballerina/build.gradle:75]: Isn't that part handle by the plugin itself ?

Comment (ayeshLK) [reply ballerina/build.gradle:75]: Fixed with [594408cc6eeaba347737dff47b2655120bc3253f](https://github.com/ballerina-platform/module-ballerinax-ibm.ctg/pull/1/commits/594408cc6eeaba347737dff47b2655120bc3253f)

File: ballerina/icon.png
Comment (ayeshLK) [reply ballerina/icon.png:1]: Fixed with [594408cc6eeaba347737dff47b2655120bc3253f](https://github.com/ballerina-platform/module-ballerinax-ibm.ctg/pull/1/commits/594408cc6eeaba347737dff47b2655120bc3253f)

File: ballerina/build.gradle (hunk 1 above)
Comment (ayeshLK) [reply ballerina/build.gradle:75]: Unless we want to do something specific (publishing native code to github packages etc) we don't have to extend the `publish` action here.

**Review by aashikam — APPROVED** [review]

**Review by Nuvindu — APPROVED** [review]

File: ballerina/Ballerina.toml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+[package]
+distribution = "2201.10.0"
+org = "ballerinax"
+name = "ibm.ctg"
+version = "0.1.0"
+license= ["Apache-2.0"]
+authors = ["Ballerina"]
+keywords = ["IBM", "CICS", "CTG", "Mainframe"]
```
Comment (NipunaRanasinghe) [inline ballerina/Ballerina.toml:8]: We usually use lowercase for the connector tags, right?

**Review by NipunaRanasinghe — APPROVED** [review]
approved with a minor suggestion

File: ballerina/Ballerina.toml (hunk 2 above)
Comment (NipunaRanasinghe) [reply ballerina/Ballerina.toml:8]: sorry it seems we have only followed that for stdlibs. so this is okay

**Pull request merged**